types/react-json-view/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for react-json-view-dl 1.21
+// Project: https://github.com/mac-s-g/react-json-view
+// Definitions by: simPod <https://github.com/simPod>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+interface Props {
+    displayObjectSize: boolean;
+    enableAdd: boolean;
+    enableEdit: boolean;
+    name: boolean;
+    src: unknown;
+    theme: string;
```

Comment:
Aren't most of those props optional? Only `src` seems to be required. Also there is more props:
https://github.com/mac-s-g/react-json-view#props

Reply:
Thank you for the review. While investigating your comment I have discovered type definitions in the package. They were just not announced in package.json.

The package does not seem to be alive much but I have created a PR there https://github.com/mac-s-g/react-json-view/pull/440